src/binding/python/Mesh.cpp

```diff
@@ -81,5 +83,6 @@ void init_Mesh(py::module &m) {
         .value("thetaMode", Mesh::Geometry::thetaMode)
         .value("cylindrical", Mesh::Geometry::cylindrical)
         .value("spherical", Mesh::Geometry::spherical)
+        .value("other", Mesh::Geometry::other)
```

Comment:
can you also add a tiny python test using this interface in `test/python/unittest/API/APITest.py`?

Reply:
done